man/man8/zpool.8

```diff
@@ -1456,7 +1456,7 @@ values, use the
 .Fl p
 flag.
 .Bl -tag -width Ds
-.It Fl c Op Ar SCRIPT1 , Ar SCRIPT2 ...
+.It Fl c Op Ar SCRIPT1 Ns Oo , Ns Ar SCRIPT2 Oc Ns ...
```

Comment:
Actually, looks as if the duplicate information was removed. the `zpool status -c` section seems to just refer back to the `zpool iostat -c` section.